juniper_official/interface/check-ifl-state-in-out-traffic.rule

```diff
@@ -0,0 +1,425 @@
+/*
+ * Monitors sub interface state and input/output traffic,it notifies when anomalies are found.
+ * Six inputs control detection
+ *
+ *   1) ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ *
+ *   2) ifl-no, is a regular expression that matches the IFL index number that
+ *      you would like to monitor.  By default it's '.*', which matches all
+ *      interfaces. Use something like '0-10*' to match only 0 to 10 IFLs.
+ *
+ *   3) in-stats-name, is a regular expression that matches the sub interface KPI
+ *      statistics counter name that you would like to monitor. By default it's
+ *      in-octets, Use proper counter name something like 'in-pkts' or
+ *      "in-unicast-pkts" or "in-multicast-pkts" etc.
+ *
+ *   4) out-stats-name, is a regular expression that matches the sub interface KPI
+ *      statistics counter name that you would like to monitor. By default it's
+ *      in-octets, Use proper counter name something like 'out-pkts' or
+ *      "out-unicast-pkts" or "out-multicast-pkts" etc.
+ * 
+ *   5) octets-high-threshold, is the threshold that causes the rule to
```

Comment:
create new field to derive mbps and modify variable name according to ifd rule.

Reply:
added field for  mbps and changed variable name